crates/router/src/connector/authorizedotnet.rs

```diff
@@ -872,11 +872,17 @@ impl api::IncomingWebhook for Authorizedotnet {
                     ),
                 ))
             }
-            _ => Ok(api_models::webhooks::ObjectReferenceId::PaymentId(
-                api_models::payments::PaymentIdType::ConnectorTransactionId(
-                    authorizedotnet::get_trans_id(&details)?,
-                ),
-            )),
+            authorizedotnet::AuthorizedotnetWebhookEvent::AuthorizationCreated
+            | authorizedotnet::AuthorizedotnetWebhookEvent::PriorAuthCapture
+            | authorizedotnet::AuthorizedotnetWebhookEvent::AuthCapCreated
+            | authorizedotnet::AuthorizedotnetWebhookEvent::CaptureCreated
+            | authorizedotnet::AuthorizedotnetWebhookEvent::VoidCreated => {
+                Ok(api_models::webhooks::ObjectReferenceId::PaymentId(
+                    api_models::payments::PaymentIdType::ConnectorTransactionId(
```

Comment:
Can we prioritize our reference in all the places?

Reply:
The connector only allows `connector_transaction_id` to be used for search on connector dashboard as well as for psync.